practice/00-HTML-CSS-basics/botom.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Form</title>
+</head>
+<body>
+<div>
+	<h1>sign up</h1>
+	<form class="contact_form">
+		<ul>
+				<li><label for="name">Name: </label><input type="text" placeholder="Michael" required></li>
+                <li><label for="name">Last Name: </label><input type="text" placeholder="Jackson" required></li>
+                <li><label for="email">Email: </label><input type="email" placeholder="name@gmail.com" required></li>
+                <li><label for="favorite sport">Favorite Sport: </label><input type="text" placeholder="For example football" required></li>
+                <li><lael for="date">Date: </label><input type="date" name="fecha"></li>
```

Comment:
You have a mistake on the label tag name, and always use english in all your code.